functions.php

```diff
@@ -7,3 +7,24 @@
  * @package Twenty Twenty-Five
  * @since Twenty Twenty-Five 1.0
  */
+
+/**
+ * Enqueue editor-style.css in the editors.
+ */
+function twenty_twenty_five_editor_style() {
+	add_editor_style( get_parent_theme_file_uri( 'editor-style.css' ) );
+}
+add_action( 'after_setup_theme', 'twenty_twenty_five_editor_style' );
+
+/**
+ * Enqueue stylesheet.css on the front.
+ */
+function twenty_twenty_five_enqueue_styles() {
+	wp_enqueue_style(
+		'twenty-twenty-five-style',
+		get_parent_theme_file_uri( 'style.css' ),
```

Comment:
I believe `get_stylesheet_uri()` should be good here.

Reply:
`get_stylesheet_uri()` gets the active themes style.css. So it depends on how and if we want the child themes to take advantage of the accessible styles.

Reply:
Perfect.

Reply:
I did not understand 😄  Should this be updated to `get_stylesheet_ur`i or not?

Reply:
@justintadlock might have some input on the use of `get_parent_theme_file_uri`?

For Twenty Twenty-Four, there was feedback from users that they wanted the style.css to be enqueued because they often make changes to it (they shouldn't, but they do...).

Reply:
@carolinan it looks good to me since you're intentionally loading the TT5 `style.css`, regardless of whether it or a child theme is active. 👍

Reply:
thank you